clients/src/main/java/org/apache/kafka/common/requests/MetadataResponse.java

```diff
@@ -431,6 +431,14 @@ public boolean isInternal() {
             return partitionMetadata;
         }
 
+        @Override
```

Comment:
This is also intentional: during trouble shooting I found that because toString is not overwritten TopicMetadata does not print correctly on log4j.